Docs/svt-hevc_encoder_user_guide.md

```diff
@@ -201,11 +201,11 @@ The resulting encoded bit stream file in binary format. If none specified, no ou
 
 >-w integer **[Required]**
 
-The width of each input image in units of picture luma pixels,  e.g. 1920
+The width of each input image in units of picture luma pixels,  e.g. 1920. This value should be a multiple of 8.
```

Comment:
No, it doesn't require to be multiple of 8.
refer to my answer:
https://github.com/OpenVisualCloud/SVT-HEVC/issues/319